src/main/kotlin/net/irisshaders/lilybot/extensions/events/LogUploading.kt

```diff
@@ -139,6 +145,9 @@ class LogUploading : Extension() {
 													// Just swallow this exception
 													// If something has gone wrong here, something is wrong
 													// somewhere else, so it's probably fine
+													//-----------------------------------------------------
```

Comment:
Why are we commenting about this and not just removing it? Or at least responding in some way and printing a stacktrace.

Reply:
> A few small tweaks. It's great that this is finally being done, thank you for that!
> 
> My main concern is the lack of proper error handling. Currently it'll just get stuck on the `Uploading <filename> to mclo.gs...` embed and it'd be amazing if that would actually change if an error occurred.

This issue is also present in the old log uploading, i don't see why you only bring it up here.
I intend on changing this in a future PR, although

Reply:
Not wanting to do it in this one is fair enough. I'll make an issue so it doesn't get lost.